examples/case_studies/item_response_nba.ipynb

```diff
@@ -0,0 +1,1142 @@
+{
```

Comment:
<p>the watermark should be:</p><pre class="ql-syntax" spellcheck="false">%load_ext watermark
%watermark -n -u -v -iv -w -p theano,xarray
</pre><p>as mentioned in the style guide that I had linked before</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>